app/components/bali/dropdown/index.scss

```
@@ -0,0 +1,5 @@
+.dropdown-component {
+  .dropdown-menu {
+    min-width: 22rem;
```

```suggestion
    min-width: max-content;
```
Talvez también debemos agregar un max-width para evitar que este demasiado largo.

Considero que los 22rem son un caso muy particular. 

¿Qué opinan?